cou/commands.py

```diff
@@ -153,6 +153,22 @@ def get_subcommand_common_opts_parser() -> argparse.ArgumentParser:
         type=batch_size_arg,
         default=1000,
     )
+    subcommand_common_opts_parser.add_argument(
```

Comment:
Without a PR description it's hard to understand what you are trying to achieve here.

As far as I understand `BooleanOptionalAction` is used to be always True and then you use the prefix `no` to turn into false. E.g: `backup` results into True and `no-backup` into `False`.

I need some more context to understand:
1 - Will this action always run and the user uses `no-purge` to not create this step?
2 - If it creates the step by explicit calling `purge`, can't we condense those two arguments to be just one? E.g: `--purge` means purge all and `--purge 2015-10-21 12:00` that means to purge before the date passed

Reply:
Because `default=False`, without `--purge` the input will be False. So no `--no-purge` is needed.

It seems we better to use `action="store_true"`

For your second approach, I think it will confuse the user. So let's keep two arguments.

Reply:
Change to `action="store_true"`